model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -68,6 +69,19 @@ data class OrtConfiguration(
      */
     val enableRepositoryPackageConfigurations: Boolean = false,
 
+    /**
+     * A list with patterns to filter out environment variables before creating child processes to prevent that those
+     * processes can access sensitive information. See [EnvironmentVariableFilter] for further details.
+     */
+    val environmentVariablesExclusionPatterns: List<String> = EnvironmentVariableFilter.DEFAULT_EXCLUSION_PATTERNS,
```

Comment:
While the "process" in `allowedProcessEnvironmentVariables` nicely mentions the contetxt in which environment variables are allowed, the `environmentVariablesExclusionPatterns` name lacks that context.

I propose `deniedProcessEnvironmentVariablePatterns` and `allowedProcessEnvironmentVariableNames` to make the variable names nicely resemble each other with "denied vs allowed" and "patterns vs names", and still naming the usage context.